_data/sites.json

```diff
@@ -3620,6 +3620,16 @@
         ]
     },
 
+    {
+        "name": "Dunkindonuts",
+        "url": "https://www.dunkindonuts.com/en/about/contact-us",
+        "difficulty": "easy",
+        "notes": "Category: DD Perks & Program -> Sub Category: Account Management for Existing Members -> 'I would like to delete my account' Message: 'Please delete all of my information for all products and services related to Dunkin' Donuts'",
```

Comment:
Is this like filling a contact form? Or does it delete immediately?

If it's immediately ok as easy, but if not we should change it to hard

Reply:
I just verified all of this on their site. It seems that DD has now changed how they do things. I need to re-do this. Yes, it was easy, but the notes need to change. Do I now make a change to my local git repo or do I change it here?